app/e2e/english_lookup_forms.spec.js

```diff
@@ -0,0 +1,144 @@
+// https://playwright.dev/docs/writing-tests#first-test
+// @ts-check
+import { test, expect } from '@playwright/test'
+
+// https://playwright.dev/docs/test-fixtures#built-in-fixtures
+test('/English/lookup/forms?word=following', async ({ request }) => {
+	const test_word = 'following'
+	const expected_forms = [
+		{
+			stem: 'following',
+			part_of_speech: 'Adjective',
+			form: 'Stem',
+		},
+		{
+			stem: 'follow',
+			part_of_speech: 'Verb',
+			form: 'Participle',
+		},
+	]
+
+	const response = await request.get(`/English/lookup/forms?word=${test_word}`)
+	const returned_forms = await response.json()
+
+	// https://playwright.dev/docs/api/class-genericassertions#generic-assertions-to-have-length
+	expect(returned_forms).toHaveLength(expected_forms.length)
+	expect(returned_forms[0]).toEqual(expected_forms[0])
+	expect(returned_forms[1]).toEqual(expected_forms[1])
+})
+
+test('/English/lookup/forms?word=accused', async ({ request }) => {
+	const test_word = 'accused'
+	const expected_forms = [
+		{
+			stem: 'accuse',
+			part_of_speech: 'Verb',
+			form: 'Past',
+		},
+		{
+			stem: 'accuse',
+			part_of_speech: 'Verb',
+			form: 'Perfect',
+		},
+	]
+
+	const response = await request.get(`/English/lookup/forms?word=${test_word}`)
+	const returned_forms = await response.json()
+
+	expect(returned_forms).toHaveLength(expected_forms.length)
+	expect(returned_forms[0]).toEqual(expected_forms[0])
+	expect(returned_forms[1]).toEqual(expected_forms[1])
+})
+
+test('/English/lookup/forms?word=got', async ({ request }) => {
+	const test_word = 'got'
+	const expected_forms = [
+		{
+			stem: 'get',
+			part_of_speech: 'Verb',
+			form: 'Past',
+		},
+		{
+			stem: 'get off',
+			part_of_speech: 'Verb',
+			form: 'Past',
+		},
+		{
+			stem: 'get up',
+			part_of_speech: 'Verb',
+			form: 'Past',
+		},
+		{
+			stem: 'get on',
+			part_of_speech: 'Verb',
+			form: 'Past',
+		},
+	]
+
+	const response = await request.get(`/English/lookup/forms?word=${test_word}`)
+	const returned_forms = await response.json()
+
+	expect(returned_forms).toHaveLength(expected_forms.length)
+	expect(returned_forms[0]).toEqual(expected_forms[0])
```

Comment:
Can't you just say `expect(returned_forms).toEqual(expected_forms)` since it does a deep equal? Or do you prefer it to be more explicit like this?

Reply:
duh... yes 🦤 

With that approach, the only thing that will change between tests is the test word and the expected results.  That makes everything much better, I'll drop the commit hash here when I finish.

Reply:
done in [5b06dcd](https://github.com/presciencelabs/tabitha-targets/pull/7/commits/5b06dcd8fbc14444b9d67a2ce18fd2eac5f873ce)

Thank you!